.gitmodules

```diff
@@ -1,18 +1,21 @@
 [submodule "third_party/eigen"]
 	path = third_party/eigen
-	url = https://github.com/PX4/eigen.git
+	url = https://github.com/swift-nav/eigen.git
```

Comment:
what is the motivation for this?  we will pick up Peddie's optimizations, but rewind a lot of upstream bug fixes.  Do we want to pull upstream into our fork...?   @anth-cole @akleeman @scarcanague 

Also, I understand that Peddie's optimizations were primarily for starling, no? Also, why are we using the PX4 fork vs. the official mirror? https://github.com/eigenteam/eigen-git-mirror

Reply:
I think we should use the same fork we use in Starling. @akleeman sound good to you?

Reply:
Yeah same as starling makes sense

Reply:
Yeah same as starling makes sense

Reply:
I made this change only to get the URLs synchronised and purposefully left the SHA the same. I'll update it so that albatross and starling point at the same commit